app/main.py

```diff
@@ -95,16 +108,66 @@ def _decrypt(self, encrypted_jwt, tx_id):
             raise RetryableError()
 
     def run(self):
+        logger.debug("Starting consumer")
         self.consumer.run()
 
     def stop(self):
+        logger.debug("Stopping consumer ")
         self.consumer.stop()
 
 
+class HealthCheck(tornado.web.RequestHandler):
+
+    @staticmethod
+    def rabbit_health():
+        http_client = HTTPClient()
+        resp = http_client.fetch(settings.RABBIT_HEALTHCHECK_URL, auth_username=settings.SEFT_RABBITMQ_MONITORING_USER,
+                                 auth_password=settings.SEFT_RABBITMQ_MONITORING_PASS)
+        body = json.loads(resp.body.decode('utf8'))
+        if body.get('status') == "ok":
+            return body.get('status')
+        return "failed"
+
+    @staticmethod
+    def ftp_health():
+        ftp = SDXFTP(logger, settings.FTP_HOST, settings.FTP_USER, settings.FTP_PASS, settings.FTP_PORT)
+        try:
+            conn = ftp._connect()
+            conn.voidcmd("NOOP")
+            return "ok"
+        except Error as e:
+            logger.debug("FTP error raised" + str(e))
+            return "failed"
+
+    def initialize(self):
+        self.get_health()
+        threading.Timer(healthcheck_delay, self.get_health()).start()
+
+    def get_health(self):
+        self.rabbit_status = self.rabbit_health()
+        self.ftp_status = self.ftp_health()
+        if self.rabbit_status == "ok" and self.ftp_status == "ok":
+            self.app_status = "ok"
+        else:
+            self.app_status = "failed"
```

Comment:
Lets use booleans rather than strings and rename "status" to healthy

ie.
"healthy":true

"healthy":false